shell/platform/windows/angle_surface_manager.cc

```diff
@@ -20,25 +20,31 @@ AngleSurfaceManager::~AngleSurfaceManager() {
   CleanUp();
 }
 
-bool AngleSurfaceManager::InitializeEGL(const EGLint* attributes) {
+static void logEglError(std::string message) {
+  EGLint error = eglGetError();
+  std::cerr << message << std::endl;
```

Comment:
You can put the `EGL: ` here before `message` instead of having it duplicated in every call site.